src/internal/connector/exchange/api/events.go

```diff
@@ -209,10 +209,11 @@ func (c Events) EnumerateContainers(
 				continue
 			}
 
-			temp := graph.NewCacheFolder(cd, path.Builder{}.Append(*cd.GetDisplayName()))
-
-			err = fn(temp)
-			if err != nil {
+			temp := graph.NewCacheFolder(
+				cd,
+				path.Builder{}.Append(*cd.GetDisplayName()),
```

Comment:
I think this should be the ID?
```suggestion
				path.Builder{}.Append(*cd.GetId()),
```

Reply:
In a future PR, yes.